src/neuroconv/nwbconverter.py

```diff
@@ -184,3 +183,42 @@ def run_conversion(
                 )
 
         return nwbfile_out
+
+
+class NWBConverterPipe(NWBConverter):
+    """Takes pre-initialized interfaces as arguments to build an NWBConverter class"""
+
+    def get_conversion_options_schema(self):
+        """Compile conversion option schemas from each of the data interface classes."""
+        conversion_options_schema = get_base_schema(
+            root=True,
+            id_="conversion_options.schema.json",
+            title="Conversion options schema",
+            description="Schema for the conversion options",
+            version="0.1.0",
+        )
+        for interface_name, data_interface in self.data_interface_classes.items():
+            conversion_options_schema["properties"].update(
+                {interface_name: unroot_schema(data_interface.get_conversion_options_schema())}
+            )
+        return conversion_options_schema
+
+    def __init__(self, data_interfaces: Union[List, Dict], verbose=True):
```

Comment:
```suggestion
    def __init__(self, data_interfaces: Union[List[BaseDataInterface], Dict[str, BaseDataInterface]], verbose=True):
```

To make content of iterables clearer